action/install.go

```diff
@@ -11,6 +11,10 @@ import (
 // Install describes an installation action
 type Install struct {
 	Driver driver.Driver // Needs to be more than a string
+
+	// OperationConfig is an optional handler that applies additional
+	// configuration to the operation before it is executed.
+	OperationConfig func(operation *driver.Operation)
```

Comment:
I suggest defining a type for these functions to keep them all in step:
```
type OperationConfigFunc func(*driver.Operation) error
```
Note that it's probably worth allowing the handler to fail. For example, it might need to apply additional configuration which somehow contradicts what's already there because some constraint has been unexpectedly broken. Returning an error seems a better approach than requiring the function to panic and presenting a stack trace to the user.

Reply:
That's a good idea, I'll make that change. 👍